typescript/infra/scripts/warp-routes/monitor-warp-routes-balances.ts

```diff
@@ -142,9 +177,21 @@ async function checkBalance(
               const balance = ethers.BigNumber.from(
                 await adapter.getBalance(token.hypAddress),
               );
-              return parseFloat(
+              const price = await tokenPriceGetter.getTokenPrice(chain);
```

Comment:
Is is safe to assume that a native token type, the token will always be the gas token of the given chain?